templates/project/_featuresList.html.twig

```diff
@@ -1,5 +1,5 @@
 
-<table class="table table-striped table-hover feature-list">
+<table class="table table-hover feature-list">
```

Comment:
reflexion plus générale (je dis pas qu'il faut le faire)
mais là ta mise en page de tableau a l'air plutot complexe. Est-ce que c'est vraiment pertienent de garder ce type de mise en page, ou partir sur des div et des flexbox ne serait-il pas plus simple ?
il sera tjs temps de modifié si c'est le cas, mais pensez y :-)

Reply:
En effet, je me suis fait la même reflexion. Mais tout refaire serait assez long et la structure for/if serait toujours là. C'est plutot qu'avec css il est plus facile de corriger l'apparence des div+flexbox que celle des tableaux.